soc/riscv/riscv-privilege/telink_b9x/start.S

```diff
@@ -62,6 +62,17 @@ start:
 	sb     t1, 0x501(t0)
 #endif
 
+	/* Disable vector mode */
+	csrci  0x7d0, 2
+	/* Disable vector feture enable address */
+#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
+	lui t0 ,0xe4000
+#elif CONFIG_SOC_RISCV_TELINK_B95
+	lui t0 ,0xc4000
+#endif
+	li  t1 ,0x00
+	sw  t1 ,0x4(t0)
```

Comment:
Now PLIC base is in t0.
Feature enable register of PLIC has offset 0.
Why are we writing at offset 4 from PLIC base?
Seems like we set to zero source 1 priority instead feature register...

Reply:
already changed , misunderstanding of the register , offset 0 means feature enable , offset 4 is the priority setting part .